src/Network/Network/ChangeLog.md

```diff
@@ -50,6 +50,7 @@
     - `New-AzApplicationGatewayFirewallPolicySetting`
 * Added optional property 'AuxiliarySku' to cmdlet `New-AzNetworkInterface` to help choose performance on an AuxiliaryMode enabled Network Interface.
 * Added a new value, 'AcceleratedConnections' for existing property 'AuxiliaryMode' for `New-AzNetworkInterface`
+* Added the option to enter an empty 'Rule' in 'New-ApplicationGatewayFirewallPolicyManagedRuleGroupOverride', which would return an empty 'RuleID' to be passed to NRP.
```

Comment:
The only change is to switch a mandatory parameter to optional, please you align the changelog

Reply:
done.